cmd/terramate/cli/cli.go

```diff
@@ -468,13 +468,13 @@ func (c *cli) printStacks() {
 		Msg("Print stacks.")
 	for _, entry := range entries {
 		stack := entry.Stack
-		stackRepr, ok := c.friendlyFmtDir(stack.Dir)
+		stackRepr, ok := c.friendlyFmtDir(stack.PrjAbsPath())
 		if !ok {
 			continue
 		}
 
 		logger.Debug().
-			Str("stack", c.wd()+stack.Dir).
+			Stringer("stack", stack).
```

Comment:
For some reason it felt like The Stringer would be a Batman villain or something :laughing: (Oh no, The Stringer !!!)

Reply:
hahahahahaha